csfieldguide/static/interactives/pixel-viewer/js/pixel-viewer.js

```diff
@@ -150,8 +150,9 @@ function setUpMode(){
   }
 }
 
```

Comment:
![image](https://user-images.githubusercontent.com/25916475/58444489-9fccaf80-814c-11e9-925a-660720a9bb24.png)
The layout screwed up (non-colour thresholding)

Reply:
It's just wrapping because of the screen size. I played around with it but it screws up the layout on the edge detection mode.